core/modules/views/plugins/views_plugin_pager_mini.inc

```diff
@@ -42,7 +42,8 @@ class views_plugin_pager_mini extends views_plugin_pager_full {
    */
   function options_form(&$form, &$form_state) {
     parent::options_form($form, $form_state);
-    unset($form['quantity']);
+
+    $options['quantity'] = NULL;
```

Comment:
Oops, that should be `$form['quantity'] = NULL;` - different variable than in function option_definition().

Yes, this change is necessary, too. I confirmed locally.

Reply:
Update: no it's not necessary (blame caching). So better revert that change.

Reply:
Done with [`1d3e9a0` (#3261)](https://github.com/backdrop/backdrop/pull/3261/commits/1d3e9a0a89f7e75d5007dfb133f1d65f5ccf75ec)